### PR TITLE
chore: minor changes to gcp cloud spanner

### PR DESCRIPTION
**Pull request by jcountsNR**
Some updates to the dashboard to add more quality utility and referencing the variable option in the description.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203252873230792

### CODE REVIEW

File: dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json [hunk 1]
```diff
@@ -2096,7 +2069,817 @@
...
+  "variables": [
+    {
+      "name": "project_id",
+      "items": null,
+      "defaultValue": {
+        "string": ""
+      },
+      "nrqlQuery": {
+        "accountIds": [
+          3462097
```
Comment (MichelLosier) [inline dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json:2847]: Looks like we have some account Ids here, mind running the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) / setting these to 0

File: dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json [hunk 2]
```diff
@@ -2096,7 +2069,817 @@
...
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "variables": [
```
Comment (MichelLosier) [inline dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json:2838]: Our team needs to investigate our ability to support dashboard variables in dashboard templates. We'll follow up once we have an answer

**Review by MichelLosier — CHANGES_REQUESTED** [review]

File: dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json (hunk 1 above)
Comment (jcountsNR) [reply dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json]: I ran the sanitize script again, so this should be cleaned up.

File: dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json (hunk 2 above)
Comment (jcountsNR) [reply dashboards/gcp-cloud-spanner/gcp-cloud-spanner.json]: I meant to remove this, thanks for spotting it. I believe supporting variables is on the backlog, so I'll wait until that's confirmed and then update this one.

**Review by MichelLosier — APPROVED** [review]
Thanks for the update!

**Pull request merged**